### DISCUSSION TITLE
Nuxt page jump without response

### DISCUSSION THREAD

**Discussion by fengluo2** [nuxt/nuxt#19575] [Questions] [answered]
After upgrading to 3.2.3,Click NuxtLink without response,It also does not report errors。
I checked the middleware，The middleware was successfully triggered
Can someone help me

**Comment by AloisSeckar** [comment]
[Reproduction](https://nuxt.com/docs/community/reporting-bugs#create-a-minimal-reproduction) would be required before someone can help you. Current description of the problem is not enough, I am afraid.

**Comment by fengluo2** [comment, marked as answer]
I found that the nuxt i18 module conflicts with him. It works normally after deleting him from my project.
